README.md

```diff
@@ -0,0 +1,129 @@
+* `dimensions` is a map of all the other dimensions that we currently care about. These fields include:
+    * `dimensions['build_version']` is the program version, like `46.0a1`.
+    * `dimensions['build_date']` is the YYYYMMDDhhmmss timestamp the program was built, like `20160123180541`. This is also known as the "build ID" or "buildid".
+    * `dimensions['channel']` is the channel, like `release` or `beta`.
+    * `dimensions['application']` is the program name, like `Firefox` or `Fennec`.
+    * `dimensions['os_name']` is the name of the OS the program is running on, like `Darwin` or `Windows_NT`.
+    * `dimensions['os_version']` is the version of the OS the program is running on.
+    * `dimensions['architecture']` is the architecture that the program was built for (not necessarily the one it is running on).
+    * `dimensions['country']` is the country code for the user (determined using geoIP), like `US` or `UK`.
+    * `dimensions['experiment_id']` is the identifier of the experiment being participated in, such as `e10s-beta46-noapz@experiments.mozilla.org`, or null if no experiment.
+    * `dimensions['experiment_branch']` is the branch of the experiment being participated in, such as `control` or `experiment`, or null if no experiment.
+    * `dimensions['e10s_enabled']` is whether E10S is enabled.
+    * All of the above fields can potentially be blank, which means "not present". That means that in the actual pings, the corresponding fields were null.
+* `stats` contains the aggregate values that we care about:
+    * `stats['usage_hours']` is the number of user-hours represented by the aggregate.
+    * `stats['main_crashes']` is the number of main process crashes represented by the aggregate (or just program crashes, in the non-E10S case).
+    * `stats['content_crashes']` is the number of content process crashes represented by the aggregate.
+    * `stats['plugin_crashes']` is the number of plugin process crashes represented by the aggregate.
+    * `stats['gmplugin_crashes']` is the number of Gecko media plugin (often abbreviated GMPlugin) process crashes represented by the aggregate.
+
+Plugin process crashes per hour on Nightly for March 14:
+
+```sql
+SELECT sum(stats['plugin_crashes'] / sum(stats->>'usage_hours') FROM aggregates
+WHERE dimensions->>'channel' = 'nightly' AND submission_date = '2016-03-14'
+```
+
+Main process crashes by build date and E10S setting.
+
+```sql
+WITH channel_rates AS (
+  SELECT dimensions['build_date'] AS build_date,
+         SUM(stats['main_crashes']) AS main_crashes, -- total number of crashes
+         SUM(stats['usage_hours']) / 1000 AS usage_kilohours, -- thousand hours of usage
+         dimensions['e10s_enabled'] AS e10s_enabled -- e10s setting
+   FROM crash_aggregates
+   WHERE dimensions['experiment_id'] = 'None' -- not in an experiment
+     AND regexp_like(dimensions['build_date'], '^\d{14}$') -- validate build IDs
```

Comment:
That depends on whether we want to include any of the pings that don't use the standard buildID form. IIRC, there's a non-trivial amount of these.